confgenerator/confgenerator.go

```diff
@@ -291,7 +283,7 @@ func getUserAgent(prefix string, hostInfo *host.InfoStat) (string, error) {
 	return expandTemplate(userAgentTemplate, prefix, extraParams)
 }
 
-func generateFluentBitInputs(receivers map[string]LoggingReceiver, pipelines map[string]*LoggingPipeline, stateDir string, hostInfo *host.InfoStat) ([]fluentbit.Input, error) {
+func generateFluentBitInputs(receivers map[string]LoggingReceiver, pipelines map[string]*LoggingPipeline, hostInfo *host.InfoStat) ([]fluentbit.Input, error) {
```

Comment:
`hostInfo` is unused here as well, though I imagine this is also going away.

Reply:
Yes, but w/e, fixed.